test/LibraryManager.IntegrationTest/AddCientSideLibrariesFromUITests.cs

```diff
@@ -0,0 +1,115 @@
+﻿using System.IO;
+using Microsoft.Test.Apex.VisualStudio.Shell.ToolWindows;
+using Microsoft.Test.Apex.VisualStudio.Solution;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Microsoft.Web.LibraryManager.IntegrationTest.Services;
+
+namespace Microsoft.Web.LibraryManager.IntegrationTest
+{
+    [TestClass]
+    public class AddCientSideLibrariesFromUITests : VisualStudioLibmanHostTest
+    {
+        ProjectTestExtension _webProject;
+        ProjectItemTestExtension _libManConfig;
+        const string _projectName = @"TestProjectCore20";
+        const string _libman = "libman.json";
+        private string _initialLibmanFileContent;
+        private string _pathToLibmanFile;
+
+        [TestInitialize]
+        public void Initialize()
+        {
+            _webProject = Solution[_projectName];
+            _libManConfig = _webProject[_libman];
+            _pathToLibmanFile = Path.Combine(SolutionRootPath, _projectName, _libman);
+            _initialLibmanFileContent = File.ReadAllText(_pathToLibmanFile);
+
+            if (File.Exists(_libManConfig.FullPath))
+            {
+                string projectPath = Path.Combine(SolutionRootPath, _projectName);
+                _libManConfig.Delete();
+                Helpers.FileIO.WaitForDeletedFile(projectPath, Path.Combine(projectPath, _libman), caseInsensitive: false);
+            }
+        }
+
+        [TestMethod]
+        public void InstallClientSideLibraries_FromProjectRoot_SmokeTest()
```

Comment:
Do you want to test install from Unpkg provider as well?

Reply:
This pr is only to test the happy path. I'll be adding more tests once these changes are checked in. 

---
In reply to: [205561546](https://github.com/aspnet/LibraryManager/pull/332#discussion_r205561546) [](ancestors = 205561546)